mod-data-import-converter-storage-server/src/main/java/org/folio/dao/AbstractProfileDao.java

```diff
@@ -174,6 +174,27 @@
     return promise.future();
   }
 
+  @Override
+  public Future<Boolean> isProfileExistById(String profileId, String tenantId) {
```

Comment:
There is already method exists with signature: public Future<Optional<T>> getProfileById(String id, String tenantId)
Can we resuse it and not create new one?

Reply:
This new method using not indexed id field from jsonb, so better to use already implemented method